sites/swarm19a.conf

```diff
@@ -75,8 +75,8 @@ server {
             rewrite ^/graphql / break;
```

Comment:
I think we'll need to remove this rewrite.

Reply:
If we remove this we'll never get to the correct end point (GET / vs POST /graphql).

Reply:
I guess I don't understand why this is needed. What is the rewrite actually intended to do? If we leave it in, Kubernetes will receive `GET /` and route the request to the wrong service.

Reply:
http://stats.zooniverse.org/graphql hits the root end point `/`
https://github.com/zooniverse/zoo-stats-api-graphql/blob/e8442c662b3112633cc3d80bdfd59b0cd94ad1b6/config/routes.rb#L6

Without this it'll try all GET requests will try to hit the non existant GET on `/graphql` https://github.com/zooniverse/zoo-stats-api-graphql/blob/e8442c662b3112633cc3d80bdfd59b0cd94ad1b6/config/routes.rb#L4

Reply:
Oh, so the graphql app doesn't respond to GET at all? That makes sense then.